e2core/server/server.go

```diff
@@ -2,25 +2,28 @@ package server
 
 import (
 	"context"
-	"net/http"
+	"fmt"
 
+	"github.com/labstack/echo/v4"
 	"github.com/pkg/errors"
-	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
+	"github.com/rs/zerolog"
+	"go.opentelemetry.io/contrib/instrumentation/github.com/labstack/echo/otelecho"
 
 	"github.com/suborbital/e2core/e2core/auth"
 	"github.com/suborbital/e2core/e2core/options"
 	"github.com/suborbital/e2core/e2core/syncer"
 	"github.com/suborbital/e2core/foundation/bus/bus"
 	"github.com/suborbital/e2core/foundation/bus/discovery/local"
 	"github.com/suborbital/e2core/foundation/bus/transport/websocket"
-	"github.com/suborbital/vektor/vk"
+	kitError "github.com/suborbital/go-kit/web/error"
+	"github.com/suborbital/go-kit/web/mid"
```

Comment:
What is `mid`? Middleware?

Reply:
Yes, that's where the common middlewares live in gokit:
* uuid request ID
* CORS
* Logger